### PR TITLE
Fix httpd.conf

### PR DESCRIPTION
**Pull request by jhadvig**
Fixing the httpd.conf so apache can serve .pl files. 
@mnagy PTAL

### CODE REVIEW

File: 5.16/contrib/httpd.conf [hunk 1]
```diff
@@ -6,12 +6,11 @@ Listen 0.0.0.0:8080
...
-    PerlResponseHandler ModPerl::Registry
-    PerlOptions +ParseHeaders
-    Options +ExecCGI
-    DirectoryIndex index.pl
-    AllowOverride All
-</Directory>
\ No newline at end of file
+PerlSwitches -I/opt/openshift/src/extlib/lib/perl5
+
+<Files ~ "\.(pl|cgi)$">
```
Comment (mnagy) [inline 5.16/contrib/httpd.conf]: Why the change from Directory?

File: 5.16/contrib/httpd.conf [hunk 2]
```diff
@@ -6,12 +6,11 @@ Listen 0.0.0.0:8080
...
-    AddHandler perl-script .pl
-    AddHandler cgi-script .cgi
-    PerlResponseHandler ModPerl::Registry
-    PerlOptions +ParseHeaders
-    Options +ExecCGI
-    DirectoryIndex index.pl
-    AllowOverride All
-</Directory>
\ No newline at end of file
+PerlSwitches -I/opt/openshift/src/extlib/lib/perl5
```
Comment (mnagy) [inline 5.16/contrib/httpd.conf]: Can't this be inside of the block bellow?

**Comment by mnagy** [comment]
I don't understand a lot of changes you made. Can you comment on why you removed/added those options? Ideally into the commit message.

File: 5.16/contrib/httpd.conf [hunk 3]
```diff
@@ -6,12 +6,11 @@ Listen 0.0.0.0:8080
...
+
+<Files ~ "\.(pl|cgi)$">
+    SetHandler perl-script
```
Comment (mnagy) [inline 5.16/contrib/httpd.conf]: Why change from AddHandler?

File: 5.16/contrib/httpd.conf (hunk 2 above)
Comment (jhadvig) [inline 5.16/contrib/httpd.conf]: No, the `PerlSwitches` directive cant be a part of any other directive (Files or Directory)

File: 5.16/contrib/httpd.conf (hunk 1 above)
Comment (jhadvig) [inline 5.16/contrib/httpd.conf]: I've realized that we dont need the `Directory` since we are simlinking the app src into the `/opt/rh/httpd24/root/var/www/html` dir

File: 5.16/contrib/httpd.conf (hunk 3 above)
Comment (jhadvig) [inline 5.16/contrib/httpd.conf]: I found this to be the proper way how to set handler for Perl scripts

**Comment by mnagy** [comment]
lgtm

File: 5.16/.sti/bin/assemble [hunk 4]
```diff
@@ -11,7 +11,7 @@ cp -Rf /tmp/src/* ./
 export CPAN_MIRROR=${CPAN_MIRROR:-""}
 
 # Don't test installed Perl modules by default
-if [ $ENABLE_CPAN_TEST = true ]; then
+if [ "$ENABLE_CPAN_TEST" = true ]; then
```
Comment (mfojtik) [inline 5.16/.sti/bin/assemble]: "${ENABLE_CPAN_TEST}"

File: 5.16/contrib/httpd.conf (hunk 2 above)
Comment (mfojtik) [inline 5.16/contrib/httpd.conf]: This should be relative or come from the template (you can't guarantee that the path will be /opt/openshift...)

**Comment by mfojtik** [comment]
2 minor nits otherwise LGTM

**Comment by jhadvig** [comment]
@mfojtik updated and rebased. Thx
